hw/ip/dcd/data/dcd.hjson

```diff
@@ -0,0 +1,402 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{ name: "dcd",
+  clock_primary: "clk_i",
+  other_clock_list: [
+    "clk_aon_i",
+  ],
+  reset_primary: "rst_ni",
+  bus_device: "tlul",
+  bus_host: "none",
+  //inter_signal_list: [
+  //  { struct: "adc_ast",
+  //    type: "req_rsp",
+  //    act: "req",
+  //    name: "adc",
+  //    package: "ast_wrapper_pkg",
+  //  }
+  //],
```

Comment:
if you have a look at `pinmux`, you'll also see how to add the "wakeup" request so that it's automatically hooked up.

Reply:
Like this?
  wakeup_list: [
    { name: "aon_wkup_req",
      desc: "pin wake request"
    }
  ],